TeamCode/src/main/java/org/firstinspires/ftc/teamcode/subsystems/chassis/Chassis.java

```diff
@@ -0,0 +1,60 @@
+package org.firstinspires.ftc.teamcode.subsystems.chassis;
+
+import com.arcrobotics.ftclib.geometry.Pose2d;
+import com.arcrobotics.ftclib.hardware.motors.Motor;
+import com.arcrobotics.ftclib.hardware.motors.MotorEx;
+import com.arcrobotics.ftclib.kinematics.wpilibkinematics.ChassisSpeeds;
+import com.arcrobotics.ftclib.kinematics.wpilibkinematics.MecanumDriveKinematics;
+import com.arcrobotics.ftclib.kinematics.wpilibkinematics.MecanumDriveWheelSpeeds;
+import com.qualcomm.robotcore.hardware.HardwareMap;
+
+import org.firstinspires.ftc.teamcode.helpers.subsystems.VLRSubsystem;
+
+public class Chassis extends VLRSubsystem<Chassis> implements ChassisConfiguration {
+    MotorEx MotorLeftFront;
+    MotorEx MotorRightFront;
+    MotorEx MotorLeftBack;
+    MotorEx MotorRightBack;
+
+    Double motorPower = 0.0;
```

Comment:
use `double` instead of `Double` where possible. types like `Double`, `Integet`, `Float` are used instead of their primitive counterparts `double`, `int`, `float` only when you need `null` as a valid value. So `Integer` can be `null,` but `int` can not.

Reply:
okay, thats good to know. thanku 🙏